packages/components/src/checkbox/docs/checkboxfield/size.tsx

```diff
@@ -0,0 +1,18 @@
+import { Checkbox, CheckboxField, Inline, Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Inline>
+            <CheckboxField size="sm">
+                <Checkbox>
+                    <Text>Option 1</Text>
+                </Checkbox>
+            </CheckboxField>
+            <CheckboxField size="md">
+                <Checkbox>
+                    <Text>Option 1</Text>
+                </Checkbox>
+            </CheckboxField>
```

Comment:
I would add a description in here.

Reply:
Why do you think we should add a description? I did not to specifically focus on the size and not add noide.